ydb/docs/en/core/deploy/configuration/config.md

```diff
@@ -558,6 +558,25 @@ Fault tolerance<br>mode | Fail<br>domain | Fail<br>realm | Number of<br>data cen
 `mirror-3-dc` | Rack | Data center | 3 | 4 in each data center | Can stand a failure of a data center and 1 rack in one of the two other data centers
 `mirror-3-dc` | Server | Data center | 3 | Doesn't matter | Can stand a failure of a data center and 1 server in one of the two other data centers
 
+## Node Broker Configuration {#node-broker-config}
+
+Node Broker is a system tablet that is responsible for registering dynamic nodes in the {{ ydb-short-name }} cluster.
+
+Node Broker can automatically generate names for dynamic nodes, which identify the node within the tenant and are displayed in metrics instead of the host name. If a dynamic node has been shut down, after a timeout its name can be taken by a new dynamic node. This allows node names to be reused when host names are changed.
+
+To enable automatic node name generation, you need to add the following to the configuration:
+
+```yaml
+feature_flags:
+  enable_dynamic_node_name_generation: true
```

Comment:
If I understand what's going on here correctly, the feature flag name is weird. It sounds like dynamic node names are always generated, but this feature flag switches the algorithm so it doesn't rely on hostnames.